### ISSUE TITLE
Icon size getting big.

### ISSUE THREAD

**Issue by KBiswas98** [codex-team/editor.js#903]
i follow all the step which is mention in this example.html (example/example.html).
but the icon getting big, i don't know how to resize it
![issu](https://user-images.githubusercontent.com/38280417/65838024-160c3900-e31c-11e9-9998-4843a7dc58a2.png)
![mm](https://user-images.githubusercontent.com/38280417/65838279-2bcf2d80-e31f-11e9-9869-9a1f56a9a429.png)

**Comment by neSpecc** [comment]
what version do you use? Is there any errors in console? Something strange with CSS, maybe your own styles overrides some styles for `svg`?

**Comment by neSpecc** [comment]
Outdated

**Comment by dipan66** [comment]
Correctly identified - I had the same issue - where the icons were getting clustered together and were big. I discovered a stray svg css inline style in the page and once I commented that - it worked well. 

I would like to request that this issue be reopened as an enhancement - for a library as good as this, it should
1. have specific classes and inline settings that enable it to not fall prey to stray css. 
2. At the very minimum - the documentation should mention this gotcha. 

IMHO - 1 should be implemented. I may have other svg on the page and wouldn't like either that or editor.js icons to be broken
